assets/src/design-system/components/menu/list/listGroupings.js

```diff
@@ -64,7 +64,7 @@ const ListGroupings = ({
               !groupOption.disabled && handleMenuItemSelect(event, groupOption)
             }
             option={groupOption}
-            listLength={listLength}
+            $listLength={listLength}
```

Comment:
FWIW this does nothing. This prop is not used anywhere.

I'm gonna remove it.

Reply:
🙃  you're right, it doesn't do anything. You can remove it. The reason why it was here is because I was combining the like 4 or 5 dropdowns used across the editor and dashboard and unifying it. At some point in my refactor I decided it was useful to know the length of the list at hand from inside of a list item, which is overridable. Since `listLength` isn't getting passed through via spreading unused props in a potential list item i decided to pass it in anyways as a transient prop so that if we needed it to add a custom style say on the final list item it was super easy.  I added the transient `$` to it here in this Pr because I was cleaning us eslint warnings on the animation dropdown. 

That's why this exists, that said - we don't need this and you are absolutely fine to remove it if you feel like it.